kubejs/server_scripts/base/recipes/lychee/block_interacting.js

```diff
@@ -279,83 +279,6 @@ ServerEvents.recipes((event) => {
         }
     ];
 
-    const ae2_conversions = [
```

Comment:
Was this added to AE2? :D

Reply:
Don't know if they've added it yet. I know they're talking about it though.